react-big-calendar/index.d.ts

```diff
@@ -26,10 +26,10 @@ declare module 'react-big-calendar' {
         step?: number;
         rtl?: boolean;
         eventPropGetter?: (event: Object, start: stringOrDate, end: stringOrDate, isSelected: boolean) => void;
-        titleAccessor?: string;
-        allDayAccessor?: boolean;
-        startAccessor?: stringOrDate;
-        endAccessor?: stringOrDate;
+        titleAccessor?: string | ((row: Object) => string);
+        allDayAccessor?: string | ((row: Object) => boolean);
+        startAccessor?: string | ((row: Object) => Date);
+        endAccessor?: string | ((row: Object) => Date);
```

Comment:
function signature is wrong, the constrains are that: "function should resolve to the specific value" and there is none other actual signature requirements, so the correct declaration should be:
`Date | (...args: any[]) => Date;`

Also note: there is no need for additional parentheses, please remove them